draft-irtf-cfrg-hpke.md

```diff
@@ -861,14 +862,15 @@ def DeriveKeyPair(ikm):
 
 where `order` is the order of the curve being used (this can be found in
 section D.1.2 of {{NISTCurves}}), and `bitmask` is defined to be 0xFF for P-256
-and P-384, and 0x01 for P-521.
+and P-384, and 0x01 for P-521. The likelihood of DeriveKeyPair failing
+with DeriveKeyPairError is 2^(-8160).
```

Comment:
I would just say "negligible", since the value varies from group to group.

If you're curious how I came to this number for P-256: the probability of one iteration failing is (2^256 - order)/2^256 = 2^-32. Then 255 iterations gives you that number.

Reply:
Oh right, thanks for this remark, I changed to “negligible”.

Thanks for the clarification on how you computed the probability.